internal/saucecloud/testcafe.go

```diff
@@ -116,12 +115,12 @@ func (r *TestcafeRunner) validateFramework(ctx context.Context, m framework.Meta
 		fmt.Print(msg.RemovalNotice(testcafe.Kind, r.Project.Testcafe.Version, r.getAvailableVersions(ctx, testcafe.Kind)))
 	}
 
-	for _, s := range r.Project.Suites {
-		if s.PlatformName != "" && !framework.HasPlatform(m, s.PlatformName) {
-			msg.LogUnsupportedPlatform(s.PlatformName, framework.PlatformNames(m.Platforms))
-			return errors.New("unsupported platform")
-		}
-	}
+	// for _, s := range r.Project.Suites {
+	// 	if s.PlatformName != "" && !framework.HasPlatform(m, s.PlatformName) {
+	// 		msg.LogUnsupportedPlatform(s.PlatformName, framework.PlatformNames(m.Platforms))
+	// 		return errors.New("unsupported platform")
+	// 	}
+	// }
```

Comment:
Here saucectl verifies that the platform specified in the suite's config is allowed according to metadata retrieved from test-composer. Commented out to force execution while testing.